packages/eyes/src/index.ts

```diff
@@ -72,21 +109,34 @@ async function waitForTestsCompletion(page: puppeteer.Page, url: string):
 
   // TODO: add a timeout for each test to make sure we terminate if they get
   // stuck.
+  let testsPassed: boolean = true;
+  consoleLog('Format: <component simName>');
   for (const [i, {title}] of tests.entries()) {
+    let result;
     await page.evaluate(`puppeteerRenderTest(${i})`);
     const screenshot = await page.screenshot();
-    // TODO: Compare the screenshot using eyes
-    console.log({title, screenshotBufferSize: screenshot.byteLength});
+    await eyes.open(projectName, title, null);
+    const {asExpected} = await eyes.checkImage(screenshot);
+    if (!asExpected && testsPassed) {
+      testsPassed = false;
+    }
+    result = await eyes.close(false);
+    logEyesResult(result);
     await page.evaluate(`puppeteerCleanupTest(${i})`);
   }
 
+  if (!testsPassed) {
```

Comment:
Setting status/quitting should be in the top level function. This function should simply return the number of failed tests.